cumulus/parachains/runtimes/assets/asset-hub-westend/src/lib.rs

```diff
@@ -825,7 +825,7 @@ impl cumulus_pallet_aura_ext::Config for Runtime {}
 
 parameter_types! {
 	/// The asset ID for the asset that we use to pay for message delivery fees.
-	pub FeeAssetId: AssetId = AssetId(xcm_config::WestendLocation::get());
+	pub FeeAssetId: AssetId = AssetId(xcm_config::AssetHubWestend::get());
```

Comment:
no change here, `WestendLocation` means `Location { parents: 1, interior: Here }` which represents relaychain native token from the parachain's perspective, which is WND (or DOT for Polkadot, KSM for Kusama...).

```suggestion
	pub FeeAssetId: AssetId = AssetId(xcm_config::WestendLocation::get());
```